docs/reference.md

```diff
@@ -1903,6 +1912,201 @@ Allowed values:
 - REJECTED
 - ALL
 
+### create-group
+
+Create a group (distribution list). _Distribution List Manager role_ is required to use this activity.
+
+Key | Type   | Required |
+------------ |--------|----------|
```

Comment:
We can sort them by "Required" (yes first and then No) :)

Reply:
Good point